packages/web3-eth/types/tests/eth.tests.ts

```diff
@@ -0,0 +1,414 @@
+
+// $ExpectType Promise<TransactionReceipt>
+eth.getTransactionReceipt('0x407d73d8a49eeb85d32cf465507dd71d507100c1');
+// $ExpectType Promise<TransactionReceipt>
+eth.getTransactionReceipt('0x407d73d8a49eeb85d32cf465507dd71d507100c1', (error: Error, transactionReceipt: TransactionReceipt) => {});
+
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1');
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1', 1000);
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1', '1000');
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1', (error: Error, count: number) => {});
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1', (error: Error, count: number) => {});
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1', 1000, (error: Error, count: number) => {});
+// $ExpectType Promise<number>
+eth.getTransactionCount('0x407d73d8a49eeb85d32cf465507dd71d507100c1', '1000', (error: Error, count: number) => {});
+
+const code = '603d80600c6000396000f3007c0';
+
+// $ExpectType PromiEvent<TransactionReceipt>
+eth.sendTransaction({
+    from: '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe',
+    data: 'code'
+});
+// $ExpectType PromiEvent<TransactionReceipt>
+eth.sendTransaction({
+    from: '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe',
+    data: 'code'
+}, (error: Error, hash: string) => {});
+
+// $ExpectType PromiEvent<TransactionReceipt>
+eth.sendSignedTransaction('0xf889808609184e72a0008227109');
+// $ExpectType PromiEvent<TransactionReceipt>
+eth.sendSignedTransaction('0xf889808609184e72a0008227109', (error: Error, gas: string) => {});
+
+// $ExpectType Promise<string>
+eth.sign('Hello world', '0x11f4d0A3c12e86B4b5F39B213F7E19D048276DAe');
+// $ExpectType Promise<string>
+eth.sign('Hello world', 3);
+// $ExpectType Promise<string>
+eth.sign('Hello world', '0x11f4d0A3c12e86B4b5F39B213F7E19D048276DAe', (error: Error, signature: string) => {});
+// $ExpectType Promise<string>
+eth.sign('Hello world', 3, (error: Error, signature: string) => {});
+
+// $ExpectType Promise<SignedTransaction>
+eth.signTransaction({
+    from: '0xEB014f8c8B418Db6b45774c326A0E64C78914dC0',
+    gasPrice: '20000000000',
+    gas: '21000',
+    to: '0x3535353535353535353535353535353535353535',
+    value: '1000000000000000000',
+    data: ''
+});
+// $ExpectType Promise<SignedTransaction>
+eth.signTransaction({
+    from: '0xEB014f8c8B418Db6b45774c326A0E64C78914dC0',
+    gasPrice: '20000000000',
+    gas: '21000',
+    to: '0x3535353535353535353535353535353535353535',
+    value: '1000000000000000000',
+    data: ''
+}, '0xEB014f8c8B418Db6b45774c326A0E64C78914dC0');
+// $ExpectType Promise<SignedTransaction>
+eth.signTransaction({
```

Comment:
missing break

Reply:
i only break after the section of tests are done, so not for every test a new break for every bunch of tests on the same method. Same style has been throughout